src/GameOptions/ui/OptionsSlider.tsx

```diff
@@ -13,14 +13,21 @@ interface IProps {
 }
 
 export const OptionsSlider = (props: IProps): React.ReactElement => {
+  const [value, setValue] = useState(props.initialValue);
+
+  const onChange = (_evt: Event, newValue: number | Array<number>): void => {
+    setValue(newValue);
+  };
+
   return (
     <Box>
       <Tooltip title={<Typography>{props.tooltip}</Typography>}>
         <Typography>{props.label}</Typography>
       </Tooltip>
       <Slider
-        value={props.value}
-        onChange={props.callback}
+        value={value}
+        onChange={onChange}
+        onChangeCommitted={props.callback}
```

Comment:
nit:
```suggestion
        onChangeEmitted={props.callback}
```

Reply:
`onChangeEmitted` isn't a part of MUI's slider API

Reply:
oh derp thought it was a custom component 👍